libraries/networking/src/NLPacket.h

```diff
@@ -82,7 +81,7 @@ class NLPacket : public udt::Packet {
     
     PacketType _type;
     PacketVersion _version;
-    QUuid _sourceID;
+    mutable QUuid _sourceID;
```

Comment:
This feels wrong to me. If we are modifying the packet, let's just remove the `const` qualifier from `sendUnreliablePacket` or pass by value. I would agree that `sendUnreliablePacket` shouldn't have to be non-`const`, but it having to be is a result of the general design. We've decided to pack all of this data together and let multiple objects modify it instead of only passing along the data that is relevant to you - ie a sender only really cares about a subset of what nlpacket provides.